cogs/vxtwitconverter/helpers.py

```diff
@@ -24,23 +52,25 @@ def convert_to_vx_twitter_url(embeds: list[Embed]):
     return vxtwitter_urls
 
 
-def urls_to_string(vx_twit_links: list[str]):
+def urls_to_string(links: list[str], socialMedia: SocialMedia):
     """
     Parameters
     ----------
-    vx_twit_links: list of urls
+    links: List[str]
```

Comment:
micronit: either just remove the type (since we have type hints already) or make it look the same as the type hint.
```suggestion
    links: list[str]
```